realm/realm-library/src/objectServer/java/io/realm/PermissionManager.java

```diff
@@ -123,10 +134,12 @@ public String getName() {
     private final Object errorLock = new Object();
     private volatile ObjectServerError permissionRealmError = null;
     private volatile ObjectServerError managementRealmError = null;
+    private volatile ObjectServerError globalPermissionRealmError = null;
 
     // Cached result of the permission query. This will be filled, once the first PermissionAsyncTask has loaded
     // the result.
-    private RealmResults<Permission> permissions;
+    private RealmResults<Permission> userPermissions;
+    private RealmResults<Permission> defaultPermissions;
```

Comment:
👍  mostly because the Realm on the server is a global Realm, shared between all users. `defaultPermissionsRealm` still sounded like it was unique to each user, but I don't have strong feelings either way. So if you prefer `defaultPermissionsRealm` I can rename it back?